pkg/cmd/controller.go

```diff
@@ -64,52 +64,71 @@ func runCostController(ko *KubeOptions, no *CostOptionsController) error {
 		return fmt.Errorf("failed to get currency code: %s", err)
 	}
 
-	if !no.isHistorical {
-		aggs, err := query.QueryAggCostModel(query.AggCostModelParameters{
-			RestConfig:        ko.restConfig,
-			Ctx:               context.Background(),
-			KubecostNamespace: *ko.configFlags.Namespace,
-			ServiceName:       no.serviceName,
-			Window:            no.window,
-			Aggregate:         "controller",
-			UseProxy:          no.useProxy,
-		})
-		if err != nil {
-			return fmt.Errorf("failed to query agg cost model: %s", err)
-		}
-
-		// don't show unallocated controller data
-		delete(aggs, "__unallocated__")
-
-		applyNamespaceFilter(aggs, no.filterNamespace)
-
-		writeAggregationRateTable(
-			ko.Out,
-			aggs,
-			[]string{"namespace", "controller"},
-			controllerTitleExtractor,
-			no.displayOptions,
-			currencyCode,
-		)
-	} else {
-		allocations, err := query.QueryAllocation(query.AllocationParameters{
-			RestConfig:        ko.restConfig,
-			Ctx:               context.Background(),
-			KubecostNamespace: *ko.configFlags.Namespace,
-			ServiceName:       no.serviceName,
-			Window:            no.window,
-			Aggregate:         "controller",
-			UseProxy:          no.useProxy,
-		})
-		if err != nil {
-			return fmt.Errorf("failed to query allocation API: %s", err)
-		}
-
-		// Use allocations[0] because the query accumulates to a single result
-		applyNamespaceFilterAllocation(allocations[0], no.filterNamespace)
-
-		writeAllocationTable(ko.Out, "Controller", allocations[0], no.displayOptions, currencyCode, true)
+	// if !no.isHistorical {
+	// aggs, err := query.QueryAggCostModel(query.AggCostModelParameters{
+	// 	RestConfig:        ko.restConfig,
+	// 	Ctx:               context.Background(),
+	// 	KubecostNamespace: *ko.configFlags.Namespace,
+	// 	ServiceName:       no.serviceName,
+	// 	Window:            no.window,
+	// 	Aggregate:         "controller",
+	// 	UseProxy:          no.useProxy,
+	// })
+	// if err != nil {
+	// 	return fmt.Errorf("failed to query agg cost model: %s", err)
+	// }
+
+	// // don't show unallocated controller data
+	// delete(aggs, "__unallocated__")
+
+	// applyNamespaceFilter(aggs, no.filterNamespace)
+
+	// writeAggregationRateTable(
+	// 	ko.Out,
+	// 	aggs,
+	// 	[]string{"namespace", "controller"},
+	// 	controllerTitleExtractor,
+	// 	no.displayOptions,
+	// 	currencyCode,
+	// )
```

Comment:
No reason to keep this around. Same thing with the rest of the commented code in this PR.